2-projects/1-todo-list/script.js

```diff
@@ -1,6 +1,53 @@
 function populateTodoList(todos) {
   let list = document.getElementById("todo-list");
   // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+
+  todos.forEach((todoItem) => {
```

Comment:
have you thought about what happens if there is a completed todo item in the array when the page loads? i.e.
```
const todos = [
  { task: "washing up", completed: true },
  { task: "walk dog", completed: false }
]
```